cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricDatasetFactory.java

```diff
@@ -157,7 +157,7 @@ private MetricsTable getOrCreateResolutionMetricsTable(String v2TableName, Strin
     return getSplitKeys(splits, splits, rowKeyDistributor);
   }
 
-  public static byte[][] getSplitKeys(int splits, int buckets, AbstractRowKeyDistributor keyDistributor) {
+  private static byte[][] getSplitKeys(int splits, int buckets, AbstractRowKeyDistributor keyDistributor) {
```

Comment:
why is splits always equal to buckets?
Do we need to pass them in as separate variables?
Or is it the case that we will always have splits==buckets for our current use case, and we may expand beyond this in the future?